plugins/Jobs.cpp

```diff
@@ -704,6 +781,11 @@ bool BedrockPlugin_Jobs::processCommand(SQLite& db, BedrockCommand& command) {
 
         const string& nextRun = result[0][1];
         const string& lastRun = result[0][2];
+        const string& state = result[0][3];
+
+        if (state == "RUNQUEUED") {
+            throw "402 Auto-retrying jobs cannot be updated once running";
```

Comment:
I am curious, why can't it be updated?

Reply:
+1, we can update regular running jobs, why wouldn't we be able to do so for ones with retryAfter?

Reply:
I have no idea.   @cead22 was this part of the original requirement?

Reply:
Yeah, but why is that? Why is it not safe for a RUNQUEUED job but it is for a RUNNING one? Maybe @quinthar knows the answer for this better

Reply:
Per https://github.com/Expensify/Bedrock/issues/111#issuecomment-327648440, I'll remove this and create a test for it tomorrow

Reply:
Actually, with this removed there is no change to `UpdateJob` so I won't create a test